src/main/scala/com/typesafe/sbt/packager/graalvm/Keys.scala

```diff
@@ -0,0 +1,12 @@
+package com.typesafe.sbt
+package packager
+package graalvm
+
+import sbt._
+
+/**
+  * GraalVM settings
+  */
+trait GraalVMKeys {
+  val nativeImageOptions = SettingKey[Seq[String]]("native-image-options", "native-image options")
```

Comment:
This is a graal only setting, right? For package specific settings we simply prefix them with the package name. In this case `graalVmNativeImageOptions` and don't scope them. This way they are clearly marked as format specific settings.

Reply:
Indeed a `native-image` specific setting.